src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/EditIssuePage.kt

```diff
@@ -49,4 +53,52 @@ class EditIssuePage(
             webElement.sendKeys(text)
         }
     }
+
+}
+
+object PageRteUtil{
+
+    private val logger: Logger = LogManager.getLogger(this::class.java)
+
+    fun clearAndTypeIfPresent(driver: WebDriver, id :String, text: String) {
```

Comment:
This one can be private. We want to only expose the RTE-compatible behavior.

Reply:
yep